config/locales/en.yml

```diff
@@ -1472,12 +1472,15 @@ en:
         total_capital:
           text: Total capital
           hint: ""
+          alt_hint: ""
         pensioner_capital_disregard:
           text: Pensioner disregard
           hint: Applied to total capital and capped at £100,000
+          alt_hint: Applied to total capital up to a maximum of £100,000
```

Comment:
This text is supposed to _replace_ the "capped at £100,000" text, not be an alternative to it. I had to clarify on the ticket with Kate, but what it's saying is if there is a non-zero pensioner disregard applied, show the new text. Otherwise don't show this line at all.